src/main/java/mate/academy/service/impl/BookServiceImpl.java

```diff
@@ -38,4 +38,23 @@ public BookDto findById(Long id) {
                 () -> new EntityNotFoundException("Can't find book by id " + id));
         return bookMapper.toDto(book);
     }
+
+    @Override
+    public void deleteById(Long id) {
+        bookRepository.deleteById(id);
+    }
+
+    @Override
+    public void setById(Long id, CreateBookRequestDto bookDto) {
+        Book beforeSet = bookRepository.findById(id).orElseThrow(
```

Comment:
also confusing name, more likely as `Set` collection
```suggestion
        Book beforeUpdate = bookRepository.findById(id).orElseThrow(
```